spec/spec_helper.rb

```diff
@@ -4,6 +4,17 @@ def load_data(path)
   File.read(File.expand_path("../data/#{path}", __FILE__))
 end
 
+def ntp_restrict_value(restrict_entry)
```

Comment:
I agree. How is it better, as a helper method o as a "let" in the specific test?